fastly-compute/geoip/fastly.toml

```diff
@@ -10,3 +10,10 @@ service_id = "CnsNqIdpSj1tZNLTl6PUi4"
 
 [scripts]
   build = "npm run build"
+
+[local_server]
+  [local_server.config_stores]
+    [local_server.config_stores.strings]
+      file = ".secret.json"
+      format = "json"
+
```

Comment:
This pulls in [config_store ](https://developer.fastly.com/reference/compute/fastly-toml/#config-stores:~:text=by%20the%20fetch.-,Config%20stores,-Test%20stores%20can)options from a seperate json file.

Reply:
This file currently lives under this repository for local development purposes, and will later be retrieved from fastly's [config store ](https://developer.fastly.com/reference/compute/fastly-toml/#config-stores:~:text=by%20the%20fetch.-,Config%20stores,-Test%20stores%20can)in production.